Doc/using/configure.rst

```diff
@@ -131,6 +131,17 @@ General Options
 
    .. versionadded:: 3.11
 
+.. cmdoption:: --enable-pystats
+
+   Turn on internal statistics gathering.
```

Comment:
Would you mind to give a more more details? In the issue, you wrote: "Currently various parts of the VM gather stats: the GC, dicts, the bytecode interpreter, type lookup cache, etc.".

Reply:
I'd rather not because, as it says, they are internal. We may well add or remove stats when we need them and this documentation will end up being a pain to maintain or just wrong.